CatFoodCalc/cat_calc.py

```diff
@@ -1,43 +1,36 @@
-class Cat:
-    # age = {
-    #     1: "kitten under 4 mo",
-    #     2: "kitten under 9 mo",
-    #     3: "kitten under 12 mo",
-    #     4: "adult cat",
-    #     5: "pregnant cat",
-    #     6: "lactating cat (under 3 kittens)",
-    #     7: "lactating cat (3+ kittens)",
-    #     8: "senior cat"
-    # }
-    # activity = {
-    #     1: "neutered, inactive and indoor cats",
-    #     2: "active and outdoor cats"
-    # }
-
-    kcal_per_kg_indoor = [52, 75]
-    kcal_per_kg_outdoor = [100]
+import presets
+
+class Cat(object):
 
     def __init__(self, weight, age, activity):
-        self.kcal_per_kg_outoor = Cat.kcal_per_kg_outdoor
-        self.kcal_per_kg_indoor = Cat.kcal_per_kg_indoor
+        self.kcal_per_kg = {
+                "outdoor": presets.kcal_per_kg["outdoor"],
+                "indoor": presents.kcal_per_kg["indoor"]
+            }
         self.weight = weight
         self.age = age
         self.activity = activity
         self.der_formula_by_weight = self.weight * 0.67
         self.der = self.calculate_der()
 
     def calculate_der(self):
-        return [round(self.der_formula_by_weight * self.get_kcal_per_kg_by_activity()[0], 2),
-                round(self.der_formula_by_weight * self.get_kcal_per_kg_by_activity()[1], 2)]
+        der = {}
+
+        amount = self.get_kcal_per_kg_by_activity()
+
+        der["min"] = self.der_formula_by_weight * amount["min"]
+        der["min"] = round(der["min"], 2)
+        
+        der["max"] = self.der_formula_by_weight * amount["max"]
+        der["max"] = round(der["max"], 2)
+
+        return(der)
 
     def get_kcal_per_kg_by_activity(self):
-        if self.activity == 2:
-            kcal_per_kg_min = self.kcal_per_kg_outoor[0]
-            kcal_per_kg_max = self.kcal_per_kg_outoor[0]
-        else:
-            kcal_per_kg_min = self.kcal_per_kg_indoor[0]
-            kcal_per_kg_max = self.kcal_per_kg_indoor[1]
-        return [kcal_per_kg_min, kcal_per_kg_max]
-
-    # def get_age_modifier(self):
-    #     # TODO
+        amount = self.kcal_per_kg[self.activity]
```

Comment:
Dictionary is the most perfect use here! That way you can select amount of kcal per kg by cat activity by using activity key. Less lines, more elegant.